stix2viz/stix2viz/stix2viz.js

```diff
@@ -788,15 +867,40 @@ function makeEdgeObject(sourceRef, targetRef, label, stixId=null)
  * @param stixObject The STIX object.  Provided in case any info from it is
  *      needed for configuring the node
  * @return A node object
+ * 
+ * Easier to work with epoch milliseconds in javascript, since Date objects
+ * don't seem to have any nice natural way to compare them.
  */
-function makeNodeObject(name, stixObject)
+
+function makeNodeObject(name, stixObject, observedDataNodes, config=null)
 {
     let node = {
         id: stixObject.get("id"),
         label: name
     };
+    let stixType = stixObject.get("type");
 
-    return node;
+    node.version = null
+    if (config)
+        // check the config first, which may contain an override of timelineTimestamps or
+        // timestamps not in timelineTimestamps
+        node.version = determineTimestampFromConfig(stixObject, stixType, config)
+
+    if (node.version === null)
+        if (stixType in timelineTimestamps)
+            // check timelineTimestamps
+            node.version = determineTimestamp(stixObject, timelineTimestamps[stixType])
+        // nothing in config or timelineTimestamps, use default behavior
+        else if (stixObject.has("modified"))
+            node.version = new Date(stixObject.get("modified")).valueOf();
+        else if (stixObject.has("created"))
+            node.version = new Date(stixObject.get("created")).valueOf();
+
```

Comment:
If the fallback is to check "modified" and then "created", why not remove those properties from the `timelineTimestamps` mapping?  These are redundant, and you could remove the entries which only listed those two properties, making it a lot shorter and simpler.

In the mapping, the "location" entry has "created" but not "modified".  Is that a bug?